dev/NumberBox/NumberBox.xaml

```diff
@@ -106,6 +106,7 @@
 
                         <TextBox x:Name="InputBox"
                             Grid.Row="1"
+                            InputScope="Number"
```

Comment:
Should this property be exposed so that developers can set it if they choose to without re-templating? I'm thinking about scenarios such as `Currency` or `TimeHour`, etc.

Reply:
As a NumberBox I don't think this really needs to be configurable. The only concern we talked through was in terms of operators +-() etc. However, those are all in the Number virtual keyboard as documented here: https://docs.microsoft.com/en-us/windows/uwp/design/input/use-input-scope-to-change-the-touch-keyboard#number

Additionally, I'm just implementing what @SavoySchuler had already speced out based on our original discussions for the NumberBox spec.

> I believe this is not a proposal, but a bug as NumberBox's spec notes here that "Number" should be the InputScope of NumberBox: https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/active/NumberBox/NumberBox.md#input-scope
https://github.com/microsoft/microsoft-ui-xaml/issues/1912#issuecomment-588003232

Reply:
I ended up going through the [InputScopeNameValue Enum](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Input.InputScopeNameValue?view=winrt-19041) and realize @karenbtlai has a good point. I somehow missed the `Currency` and `TimeHour` examples in her comment so wasn't thinking this through as deeply. That said, I think the actual virtual keyboard doesn't have all these cases implemented separately so value added at this point is unknown.

Entries that require numerical-only input:
`CurrencyAmount`, `CurrencyAmountAndSymbol`, `DateDayNumber`, `DateMonthNumber`, `DateYear`, `TimeHour`, `TimeMinutesOrSeconds`, `Digits`, `Number`, `NumberFullWidth`, `NumericPassword`, `NumericPin`, `TelephoneAreaCode`, `TelephoneCountryCode`, `TelephoneLocalNumber`, `TelephoneNumber`

`Formula` and `FormulaNumber` may be needed in the future if the forumla is preserved and not immediately evaluated by the number box.

The telephone input scopes would be a lot more useful if we had formatted supported. But we should still support that as hopefully it's added in the future.

I'll wait for additional feedback but am leaning towards making this a DP now.

Reply:
Thanks for the analysis @SavoySchuler FYI.

Reply:
I believe the consensus is that this should be exposed as an API on numberbox with this value as the default. I opened a new issue so I'm going to merge this in.  @robloo if your interested it be awesome if you could make the API update too! :) #2710

Reply:
Sure, I'll add the property and update the API!